nautobot/docs/release-notes/version-2.0.md

```diff
@@ -10,6 +10,10 @@ If you are a user migrating from Nautobot v1.X, please refer to the ["Upgrading
 
 ### Added
 
+#### Generic Role Model ([#1063](https://github.com/nautobot/nautobot/issues/1063))
+
+DeviceRole, RackRole, IPAM Role, and IPAddressRoleChoices have all been merged into a single generic Role model. A role can now be created and associated to one or more of the content-types that previously implemented role as a field. These model content-types include dcim.device, dcim.rack, virtualization.virtualmachine, ipam.ipaddress, ipam.prefix, and ipam.vlan.
+
 #### Added Site Fields to Location ([#2954](https://github.com/nautobot/nautobot/issues/2954))
 
```

Comment:
Need an item under `### Changed` as well to indicate that the IPAddress.role field has been changed in its behavior.

Also should probably revise the role-related content in `nautobot/docs/models/ipam/ipaddress.md`.